services/ui-src/src/components/fields/ChoiceListField.test.tsx

```diff
@@ -95,6 +95,7 @@ describe("Test ChoiceListField component rendering", () => {
 
   it("RadioField should render nested child fields for choices with children", () => {
     // Render Initial State and choices
+    mockGetValues(undefined);
```

Comment:
Fixes `Warning: Failed prop type: Invalid prop 'value' supplied to 'TextInput', expected one of type [string, number].`